lib/src/builder/parsing/member_association.dart

```diff
@@ -159,6 +160,9 @@ String getRelatedName(BoilerplateMember member) {
   return null;
 }
 
-extension<T> on T {
-  S mapIfNotNull<S>(S Function(T) callback) => this == null ? null : callback(this);
+extension<T> on T? {
+  S? mapIfNotNull<S>(S? Function(T) callback) {
+    final self = this;
+    return self == null ? null : callback(self);
+  }
```

Comment:
![..isFancy=true](https://media1.giphy.com/media/147p93s2vFKsQU/giphy.gif)

Reply:
you fancy Greg :)

Reply:
Heh thanks; there may have been a less "clever" way to write this code, but this utility helped quite a few places in this file not be so verbose